### PR TITLE
rebrand

### PR DESCRIPTION
**Pull request by daniel-house**
@madolson @zuiderkwast @stockholmux please review

### CODE REVIEW

File: docs/management/security/_index.md [hunk 1]
```diff
@@ -159,19 +159,19 @@ While it would be a strange use case, the application should avoid composing the
...
-to random paths. This is [a security issue](http://antirez.com/news/96) that may lead to the ability to compromise the system and/or run untrusted code as the same user as Redis is running.
+the name of the dump file. This allows clients to write RDB Valkey files
+to random paths. This is [a security issue](http://antirez.com/news/96) that may lead to the ability to compromise the system and/or run untrusted code as the same user as Valkey is running.
 
-Redis does not require root privileges to run. It is recommended to
-run it as an unprivileged *redis* user that is only used for this purpose.
+Valkey does not require root privileges to run. It is recommended to
+run it as an unprivileged *valkey* user that is only used for this purpose.
 
 ## GPG key
```
Comment (zuiderkwast) [inline docs/management/security/_index.md:176]: Let's delete this GPG key. If anyone sends us emails with this one, we can't decrypt them.

**Review by zuiderkwast — COMMENTED** [review]
Good.

Shall we write our security email address somewhere here. We have one now i think.

File: docs/management/security/_index.md [hunk 2]
```diff
@@ -11,116 +11,116 @@ aliases: [
 ---
 
 This document provides an introduction to the topic of security from the point of
-view of Redis. It covers the access control provided by Redis, code security concerns,
+view of Valkey. It covers the access control provided by Valkey, code security concerns,
 attacks that can be triggered from the outside by selecting malicious inputs, and
 other similar topics. 
-You can learn more about access control, data protection and encryption, secure Redis architectures, and secure deployment techniques by taking the [Redis University security course](https://university.redis.com/courses/ru330/).
+You can learn more about access control, data protection and encryption, secure Valkey architectures, and secure deployment techniques by taking the PLEASE-REVIEW{[Redis University security course](https://university.redis.com/courses/ru330/)}.
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/security/_index.md]: done

File: docs/management/security/_index.md [hunk 3]
```diff
@@ -11,116 +11,116 @@ aliases: [
...
+by unprotected Valkey instances executed without proper administration. However,
+the system administrator can still ignore the error given by Valkey and
 disable protected mode or manually bind all the interfaces.
 
 ## Authentication
 
-Redis provides two ways to authenticate clients.
-The recommended authentication method, introduced in Redis 6, is via Access Control Lists, allowing named users to be created and assigned fine-grained permissions.
+Valkey provides two ways to authenticate clients.
+The recommended authentication method PLEASE-REVIEW{, introduced in Valkey 6,} is via Access Control Lists, allowing named users to be created and assigned fine-grained permissions.
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/security/_index.md]: done

File: docs/management/security/_index.md (hunk 1 above)
Comment (daniel-house) [reply docs/management/security/_index.md]: done

File: docs/management/security/_index.md [hunk 4]
```diff
@@ -11,116 +11,116 @@ aliases: [
...
 
-It is possible to disallow commands in Redis or to rename them as an unguessable
+It is possible to disallow commands in Valkey or to rename them as an unguessable
 name, so that normal clients are limited to a specified set of commands.
 
-For instance, a virtualized server provider may offer a managed Redis instance
+For instance, a virtualized server provider may offer a managed Valkey instance
 service. In this context, normal users should probably not be able to
-call the Redis **CONFIG** command to alter the configuration of the instance,
+call the Valkey **CONFIG** command to alter the configuration of the instance,
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/security/_index.md]: done

File: docs/management/security/_index.md [hunk 5]
```diff
@@ -11,116 +11,116 @@ aliases: [
...
+Failing to protect the Valkey port from the outside can have a big security
+impact because of the nature of Valkey. For instance, a single `FLUSHALL` command can be used by an external attacker to delete the whole data set.
 
 ## Protected mode
 
-Unfortunately, many users fail to protect Redis instances from being accessed
+Unfortunately, many users fail to protect Valkey instances from being accessed
 from external networks. Many instances are simply left exposed on the
-internet with public IPs. Since version 3.2.0, Redis enters a special mode called **protected mode** when it is
+internet with public IPs. PLEASE-REVIEW{Since version 3.2.0,} Valkey enters a special mode called **protected mode** when it is
```
[the remark this answers is not in the document]
Comment (daniel-house) [reply docs/management/security/_index.md]: done

**Review by zuiderkwast — APPROVED** [review]

**Pull request merged**